modules/tasks/tasks/src/test/java/org/motechproject/tasks/service/TaskTriggerHandlerTest.java

```diff
@@ -1721,6 +1713,18 @@ private MotechEvent createEvent(boolean withCustomParser) {
         return map;
     }
 
+    private void prepareActivityServiceAnswer() {
+        doAnswer(new Answer<Void>() {
```

Comment:
Instead of doing this, create a new hashmap in handleError in TaskTriggerHandler. In general its better not to mix these maps